### PR TITLE
Add IP blocks from RFC 1918

### PR DESCRIPTION
**Pull request by mihaileu**
adding missing private(local) ip block

### CODE REVIEW

**Comment by pimjansen** [comment]
Please indicate what you want to achieve with the change and ensure that only the relevant changes are in the PR

File: src/Faker/Provider/Internet.php [hunk 1]
```diff
@@ -45,23 +45,23 @@ public function email()
      */
     final public function safeEmail()
     {
-        return preg_replace('/\s/u', '', $this->userName() . '@' . static::safeEmailDomain());
+        return preg_replace('/\s/u', '', $this->userName().'@'.static::safeEmailDomain());
```
Comment (chris-doehring) [inline src/Faker/Provider/Internet.php:48]: Can you please add those spaces again for the concatenations? There is no defined PSR standard for concatenations, but keeping spaces around them improves readability. However, I agree to your other changes with short array syntax and additions of spaces.

File: src/Faker/Provider/Internet.php [hunk 2]
```diff
@@ -204,21 +206,23 @@ public function ipv6()
      */
     public static function localIpv4()
     {
-        if (static::numberBetween(0, 1) === 0) {
-            // 10.x.x.x range
-            return long2ip(static::numberBetween(ip2long("10.0.0.0"), ip2long("10.255.255.255")));
-        }
+        $ipBlocks = [
```
Comment (chris-doehring) [inline src/Faker/Provider/Internet.php:209]: Can you please add a reference to the corresponding RFC document? (https://tools.ietf.org/html/rfc1918#section-3)

File: src/Faker/Provider/Internet.php (hunk 1 above)
Comment (mihaileu) [reply src/Faker/Provider/Internet.php:48]: this is why style ci failed, i followed https://github.styleci.io/analyses/nN1MxO?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link which told me to do those changes. Who should I follow, you or style ci ?

Comment (chris-doehring) [reply src/Faker/Provider/Internet.php:48]: If style CI is suggesting that, I give way to the CI :)

**Comment by Nyholm** [comment]
Could you please revert the unrelated CS changes.

File: src/Faker/Provider/Internet.php [hunk 3]
```diff
@@ -247,112 +251,112 @@ protected static function toAscii($string)
...
-                'ắ'=>'a', 'ş'=>'s', "'"=>'', 'ու'=>'u', 'ա'=>'a', 'բ'=>'b',
-                'գ'=>'g', 'դ'=>'d', 'ե'=>'e', 'զ'=>'z', 'է'=>'e', 'ը'=>'y',
-                'թ'=>'t', 'ժ'=>'zh', 'ի'=>'i', 'լ'=>'l', 'խ'=>'kh', 'ծ'=>'ts',
-                'կ'=>'k', 'հ'=>'h', 'ձ'=>'dz', 'ղ'=>'gh', 'ճ'=>'ch', 'մ'=>'m',
-                'յ'=>'y', 'ն'=>'n', 'շ'=>'sh', 'ո'=>'o', 'չ'=>'ch', 'պ'=>'p',
-                'ջ'=>'j', 'ռ'=>'r', 'ս'=>'s', 'վ'=>'v', 'տ'=>'t', 'ր'=>'r',
-                'ց'=>'ts', 'փ'=>'p', 'ք'=>'q', 'և'=>'ev', 'օ'=>'o', 'ֆ'=>'f',
-            );
+            $transliterationTable = [
+                'Ĳ'=> 'I', 'Ö'=>'O', 'Œ'=>'O', 'Ü'=>'U', 'ä'=>'a', 'æ'=>'a',
```
Comment (SeanJA) [inline src/Faker/Provider/Internet.php:255]: why did you add a space to only the first thing in the array on each line?
